### ISSUE TITLE
After building the project, the qt folder do not have configure.exe

### ISSUE THREAD

**Issue by anuragaj** [wkhtmltopdf/wkhtmltopdf#2358]
I tried to build the project python build.py msvc2013-win32, everything goes fine but, the qt folder : static-build\...\qt do not have configure.exe.

**Comment by jhackney** [comment]
Hi, how did you resolve your issue?  I have the same problem.  Thanks.

**Comment by ashkulz** [comment]
What **is** the problem?

**Comment by anuragaj** [comment]
@jhackney do you have 'qt' project in the main directory (along with 'scripts', 'src' and others) ?

**Comment by jhackney** [comment]
Hi Anurag,

Thanks for responding to me.  Later yesterday I figured out that the 'qt'
project was missing.  When I downloaded the zip file from GIT, it didn't
include the contents of the qt folder.  I'm new to this project so it took
me some time to figure out what it needed.

Thanks again,
Jonas

On Tue, Jun 2, 2015 at 9:59 AM, Anurag Joshi <notifications@github.com>
wrote:

> @jhackney <https://github.com/jhackney> do you have 'qt' project in the
> main directory (along with 'scripts', 'src' and others) ?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/wkhtmltopdf/wkhtmltopdf/issues/2358#issuecomment-107965061>
> .
>



-- 
Thanks,
Jonas Hackney
